src/seam-os/routes.ts

```diff
@@ -2,16 +2,73 @@
 // `yarn build:route-types` to update it
 // TODO automatically pull this from @seamapi/public-types
 export interface Routes {
-  "/health": {
-    route: "/health"
+  "/api/health": {
```

Comment:
I think there might be an issue with nextlove, these routes shouldn't be prefixed with `/api`

Reply:
oop!